src/components/App.jsx

```diff
@@ -9,12 +11,27 @@ export default class App extends Component {
   state = {
     sideBarOpened: $(window).width() < 600 ? false : true,
     SearchLoading: false,
+    dialogOpen: true
```

Comment:
Since you have older state named `sideBarOpened`, probably `dialogOpened` is more correct compared to `dialogOpen`.
On the other hand dialogOpen is a very general name for this specific state.